scheduler/src/manager.rs

```diff
@@ -68,7 +65,9 @@ impl Manager {
     /// A JoinHandle<()>
     fn create_grpc_server(&self, tx: mpsc::Sender<Event>) -> JoinHandle<()> {
         info!("creating grpc server ...");
-        let addr = "127.0.0.1:50051".parse().unwrap();
+        let addr = format!("{}:{}", self.config.grpc.host, self.config.grpc.port)
+            .parse()
+            .unwrap();
```

Comment:
No unwrap, please. `create_grpc_server`should return a `Result<JoinHandle<()>>`

Reply:
Fix.